pkg/transformers/image.go

```diff
@@ -76,8 +76,7 @@ func (pt *imageTransformer) findAndReplaceImage(obj map[string]interface{}) erro
 	for _, path := range paths {
 		containers, found := obj[path]
 		if found {
-			_, err := pt.updateContainers(containers)
-			if err != nil {
+			if _, err := pt.updateContainers(containers); err != nil {
```

Comment:
Thanks for taking this on. I was initially surprised that this transformer would cause issue #890. Is there a reason why the search path in line 74 can not be made more specific?

Reply:
It seems originally authored by @narg95 .

It is kept for backward compatibility. There was some discussion in #841 for your information.